plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiConnectorTest.java

```diff
@@ -25,13 +25,15 @@
 public class TestHudiConnectorTest
         extends BaseConnectorTest
 {
+    private static final long COMMIT_TIMESTAMP = 20251027183851494L;
```

Comment:
Is the naming aligned with the Hudi glossary? using "commit timestamp" sounds more like a temporal read, it actually more like a "commit version"